settings.py

```diff
@@ -2127,3 +2127,5 @@ def _pkce_required(client_id):
 # make sure you add it before localsettings are imported (from localsettings import *)
 
 MAX_GEOSPATIAL_INDEX_DOC_LIMIT = 1000000
+
+APP_INTEGRITY_SAMPLE_PERCENT = 0.1  # % of users
```

Comment:
If this were a model (or a release toggle?), we could modify the percentage without a deploy, using the django admin (or toggles page). That might make it easier to fine tune the percentage over time, in case we miscalibrated.